spring-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+hazelcast.viridian.url=https://dev.test.hazelcast.cloud
```

Comment:
Will we need the URL for production? Doesn't the client default to the production cloud environment?

Reply:
that would be `coordinator.hazelcast.com` - which would be different again on the new environment 
Yes. the client defaults to production environment

Reply:
I think to avoid confusing customers/users, we should remove this.